submissions/bolarinwa_fasanmi/HTML/landingpage.html

```diff
@@ -0,0 +1,146 @@
+                <h2>Popular Deals</h2> <br>
+                <div class="btn">
+                    <button onclick="loadproducts()">view more products</button>
+                </div>
+                
+                <div class="product">
+                    <div class="product_box">
+                        <div id="product_image">
+                            <img src="image/fitnesswatch.jpg" alt="fitness_watch" class="image"/>
+                        </div>
+                        <div class="product_name">
+                            Fitness watch
+                        </div>
+                        <div class="product_price">
+                            N7,000
+                        </div>
+                        
+                    </div>
+
+                    <div class="product_box">
+                        <div id="product_image">
+                            <img id="product_image" src="image/smartwatch.jpg" alt="smart_watch" class="image"/>
+                        </div>
+                        <div class="product_name">
+                            Smart watch
+                        </div>
+                        <div class="product_price">
+                            N5,000
+                        </div>
+                        
+                        
+                    </div>
+
+                    <div class="product_box">
+                        <div id="product_image">
+                            <img src="image/formalshoes.jpg" alt="formal_shoes" class="image">
+                        </div>
+                        <div class="product_name">
+                            Formal shoes
+                        </div>
+                        <div class="product_price">
+                            N15,000
+                        </div>
+                        
+                    </div>
+
+                    <div class="product_box">
+                        <div id="product_image">
+                            <img src="image/bags.jpg" alt="bags" class="image">
+                        </div>
+                        <div class="product_name">
+                            Bags
+                        </div>
+                        <div class="product_price">
+                            N5,500
+                        </div>
+                    </div> <br>
+                    <div class="product_box">
+                           <div id="product_img">
+                                <img src="image/t-shirt.jpg" alt="">
+                                    </div>
+                                    <div class="product_name">
+                                        T-shirt
+                                    </div>
+                                    <div class="product_price">
+                                        N6,000
+                                    </div> 
+                   
+                    
+                    
+                    
+                </div>
+

```

Comment:
There is supposed to be a closing _div_ tag here for the _div class="section"_ listed above.